data/json/recipes/tools/tool.json

```diff
@@ -955,6 +975,20 @@
     "using": [ [ "plastic_molding", 1 ] ],
     "components": [ [ [ "plastic_chunk", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "fake",
+    "result": "gasket_rubber",
```

Comment:
I think it would be better to remove vacuum molding requirements from the plastic gasket recipe, than to add an alternative recipe. Also this recipe doesn't currently spawn anywhere.